src/lib/up-storage.ts

```diff
@@ -227,11 +228,6 @@ class ProxyStorage implements IProxy {
       agentOptions: this.agent_options,
     };
 
-    // GET requests should not have a body, otherwise the request might fail(return 413 status code)
-    if (method.toUpperCase() !== 'GET') {
-      requestOptions.body = json
-    }
-
```

Comment:
I deleted these code, because I found the bug is in [this line](https://github.com/verdaccio/verdaccio/blob/bae430fe24c9bb63dcf7b7b9e5f9f0dbf3f42669/src/lib/up-storage.ts#L157).
I create a new [PR](https://github.com/verdaccio/verdaccio/pull/3647) to fix this bug.

> don't forget update `verdaccio/core`